build.sbt

```diff
@@ -40,6 +40,9 @@ lazy val freestyle = (crossProject in file("freestyle")).
     libraryDependencies ++= Seq(
       "org.typelevel" %%% "cats-free" % "0.9.0",
       "org.scala-lang" % "scala-reflect" % scalaVersion.value,
+      "io.monix" %%% "monix-eval" % "2.2.1" % "test",
+      "io.monix" %%% "monix-cats" % "2.2.1" % "test",
```

Comment:
Should we move the monix test to `freestyle-monix`?
That way we wouldn't need monix as a test dependency in the core  `freestyle` module?